docs/libraries.md

```diff
@@ -0,0 +1,77 @@
+---
+id: libraries
+title: Using Libraries
+author: Brent Vatne
+authorURL: https://twitter.com/notbrent
+description: This guide introduces React Native developers to finding, installing, and using third-party libraries in their apps.
+---
+
+React Native provides a set of built-in [Core Components and APIs](./components-and-apis) ready to use in your app. You're not limited to the components and APIs bundled with React Native. React Native has a community of thousands of developers. If the Core Components and APIs don't have what you are looking for, you may be able to find and install a library from the community to add the functionality to your app.
+
+## Selecting a Package Manager
+
+React Native libraries are typically installed from the [npm registry](https://www.npmjs.com/) using a Node.js package manager such as [npm CLI](https://docs.npmjs.com/cli/npm) or [Yarn Classic](https://classic.yarnpkg.com/).
```

Comment:
"Yarn Classic" link is not working (too many redirects). It looks like `https://classic.yarnpkg.com/lang/en/` is the correct URL.

BTW. Are there any specific reasons why the v1 is recommended?

Reply:
Yarn v2 installs packages without `node_modules`, which isn't compatible with some prior code that made assumptions about `node_modules`, as you might expect. https://github.com/yarnpkg/berry/issues/766

Reply:
@Simek - that's strange - what browser are you using? might also be a locale issue, I used the bare https://classic.yarnpkg.com/ url so it can redirect to the appropriate language, eg: for me https://classic.yarnpkg.com/lang/en/ - but we could just use the english language link if that's causing issues for people.

Reply:
@ide Thank you for the explanation and the link. But to be 100% correct, according to the issue you provided and the last post which summarized the whole discussion, V2 supports installation of `node_modules` but it requires the additional configuration file (`.yarnrc.yml`). So it's reasonable until the whole situation settles and the V2 stables to recommend using the V1.

@brentvatne Clearing the local cache solved the "Too many redirects" issue, but the redirect is still there. Since language select is available from the top menu and until RN website doesn't support i18n I think that it's safer to use the full link to English version to avoid similar issues brought by the community in the future.

Reply:
thanks for checking @Simek! I'll update to point to https://classic.yarnpkg.com/en/ then